build.sh

```diff
@@ -11,6 +11,7 @@
 
 base=$(pwd)
 
+GOCMD=${GOCMD:-"go"}
```

Comment:
Maybe we should make this default to go1.15 since that's what our go.sum is based on?

Reply:
that's fair